File: C-Sharp/LBI.cs
using CrystalDecisions.CrystalReports.Engine;
using CrystalDecisions.Windows.Forms;
using DpSdkEngLib;
using DPSDKOPSLib;
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
namespace _4PosBackOffice.NET
{
	public class LBI
	{
		private string mString;

		private object mT;
		public LBI(string inString, object inT)
		{
			mString = inString;
			mT = inT;
		}

		public override string ToString()
		{
			return mString + ", " + mT;
		}

	}
}

File: VB/4PosBackOffice.NET/LBI.vb
Public Class LBI
    Private mString As String
    Private mT As Object

    Public Sub New(inString As String, inT As Object)
        mString = inString
        mT = inT
    End Sub

    Public Overrides Function ToString() As String
        Return mString & ", " & mT
    End Function

End Class
